src/components/weather-page/SingleDay.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {SingleDayData} from './SingleDayData';
+import './singleDay.scss';
+
+import ThermometerIcon from './resources/svg_icon_thermometer';
+import CloudIcon from './resources/svg_icon_cloud';
+import SunIcon from './resources/svg_icon_sun';
+import WindIcon from './resources/svg_icon_wind';
+
+// TODO - Add Image Array & random picker function for weather card icon
+// TODO - Add function to convert "2019-11-14T16:08:54Z" into weekday i.e. Monday
+// TODO - Generate weather text summary other than lorem ipsum
```

Comment:
Probably best to use JIRA cards for things like this, rather than in code TODOs.

That way it can all be easily scheduled in.